trace-sdk/src/main/java/io/bitrise/trace/scheduler/ServiceScheduler.java

```diff
@@ -29,6 +30,10 @@
   @NonNull
   private final Class<? extends JobService> jobService;
 
+  @VisibleForTesting
+  @NonNull
+  ComponentName componentName;
```

Comment:
this ComponentName caused issues whilst testing - I've made a class property so that I can override it in the test.